src/utils/issue.ts

```diff
@@ -11,13 +11,20 @@ export function isParentIssue(body: string) {
   return body.match(parentPattern);
 }
 
-export async function getAssignedIssues(context: Context, username: string) {
-  const payload = context.payload;
+export async function getAssignedIssues(context: Context, username: string): Promise<GitHubIssueSearch["items"]> {
+  let repoOrgQuery = "";
+  if (context.config.checkAssignedIssues === "repo") {
+    repoOrgQuery = `repo:${context.payload.repository.full_name}`;
+  } else {
+    context.organizations.forEach((org) => {
+      repoOrgQuery += `org:${org} `;
+    });
+  }
 
   try {
-    return await context.octokit
+    return context.octokit
       .paginate(context.octokit.rest.search.issuesAndPullRequests, {
-        q: `org:${payload.repository.owner.login} assignee:${username} is:open is:issue`,
+        q: `${repoOrgQuery} assignee:${username} is:open is:issue`,
```

Comment:
Since you use the organizations with a `search`, it is possible that the user has a private profile which will result in a `422` error when querying the search API. You need to provide a fallback for this.

Reply:
When I tested with a user having a private profile, the run crashed on a 422 error. I will test again.